pages/linux/blurlock.md

```diff
@@ -0,0 +1,21 @@
+# blurlock
+
+> A simple wrapper around the i3 screen locker `i3lock`, which blurs the screen.
+> See also `i3lock`.
```

Comment:
```suggestion
> See also: `i3lock`.
```

Reply:
We use `See also ` 87 times and `See also:` 51 times. We should standardize that. Not sure to what, but I'll just apply your suggestion for now.